styles/app.css

```diff
@@ -0,0 +1,21 @@
+@import url('vendor/bootstrap.min.css');
```

Comment:
I prefer the solution with bootstrap in html page (for the purpose of the course)